src/main/scala/Garage/Garage.scala

```diff
@@ -25,28 +25,60 @@ class Garage {
   }
 
 
-  def registerEmployee: Unit = {
-
+  def registerEmployee(employeeToBeRegistered: Employee): Unit = {
+    listOfEmployees = listOfEmployees :+ employeeToBeRegistered
   }
 
-  def fixVehicle: Unit = {
+  def fixVehicle(vehicleToBeFixed: Vehicle, currentTotalEmployeeWorkTime: Int, workableEmployees: Int): Int = {
```

Comment:
there's way too many things happening in this method, can you split it up into multiple methods and just call them?